tests/modules/nf-core/endorspy/test.yml

```diff
@@ -1,7 +1,85 @@
-- name: endorspy test_endorspy
-  command: nextflow run ./tests/modules/nf-core/endorspy -entry test_endorspy -c ./tests/config/nextflow.config  -c ./tests/modules/nf-core/endorspy/nextflow.config
+- name: endorspy test_endorspy_raw
+  command: nextflow run ./tests/modules/nf-core/endorspy -entry test_endorspy_raw -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/endorspy/nextflow.config
   tags:
-    - endorspy
+  - endorspy
   files:
-    - path: output/endorspy/test_endogenous_dna_mqc.json
-      md5sum: fe7bea111d966db321dcde7fbfb062b1
+  - path: output/endorspy/test_percent_on_target_mqc.json
+    md5sum: 4f161c6bbcaac68a24b83628e2072777
+  - path: output/samtools/test.flagstat
+    md5sum: 4f7ffd1e6a5e85524d443209ac97d783
+
+- name: endorspy test_endorspy_raw_qualityfilter
+  command: nextflow run ./tests/modules/nf-core/endorspy -entry test_endorspy_raw_qualityfilter -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/endorspy/nextflow.config
+  tags:
+  - endorspy
+  files:
+  - path: output/endorspy/test_percent_on_target_mqc.json
+    md5sum: 1940461b14e6150138ca71ef7d395440
+  - path: output/samtools/test.bam
+    md5sum: 1bfa483a2991037107732ccedef8ae07
+  - path: output/samtools/test.bam.bai
+    md5sum: 077fa5c7206bf5cd43a08788c79ab9f2
+  - path: output/samtools/test.filtered.flagstat
+    md5sum: eed2b068e1eb7a3d748f094738e26ad0
+  - path: output/samtools/test.flagstat
+    md5sum: 4f7ffd1e6a5e85524d443209ac97d783
```

Comment:
All of these are not necessary, you just need to check for the JSOn (with md5sum) and the `versions.yml` (no md5sum check, just the path)

Reply:
would I just remove this?